README.md

```diff
@@ -1,2 +1,62 @@
 # Hello_Microverse
 Hello World
+
+
```

Comment:
Kindly provide a detailed description of your project here. Remember that a detailed description of your project in the readme file always make your project looks proffessional and also help others know what the project is all about before even delving into your code. :+1: